### PR TITLE
push all calls to up and down to look in bin

### PR DESCRIPTION
**Pull request by coffeepac**
this is a cleanup from samsung-cnct/k2#718

### CODE REVIEW

File: tests/aws-testing.sh [hunk 1]
```diff
@@ -15,22 +15,22 @@ echo "modify config in-place"
 build-scripts/update-generated-config.sh cluster/aws/config.yaml ${JOB_BASE_NAME}-${BUILD_ID}
 
 echo "dry run"
-PWD=`pwd` && ./up.sh --config $PWD/cluster/aws/config.yaml --output $PWD/cluster/aws/ -t dryrun
+PWD=`pwd` && bin/up.sh --config $PWD/cluster/aws/config.yaml --output $PWD/cluster/aws/ -t dryrun
```
Comment (guineveresaenger) [inline tests/aws-testing.sh]: This should be ./bin/up.sh etc...

File: tests/aws-testing.sh [hunk 2]
```diff
@@ -15,22 +15,22 @@ echo "modify config in-place"
...
 if [ $? -ne 0 ]; then
   echo "Failed at dry run step"
   exit 1;
 fi
 
 echo "up and down"
 err=0
-PWD=`pwd` && ./up.sh --config $PWD/cluster/aws/config.yaml --output $PWD/cluster/aws/
+PWD=`pwd` && bin/up.sh --config $PWD/cluster/aws/config.yaml --output $PWD/cluster/aws/
```
Comment (guineveresaenger) [inline tests/aws-testing.sh]: ./bin/up.sh

File: tests/aws-testing.sh [hunk 3]
```diff
@@ -15,22 +15,22 @@ echo "modify config in-place"
...
 up_err=$?
 if [ $up_err -ne 0 ]; then
   err=$up_err
   echo "./up.sh failed"
 fi
 
-PWD=`pwd` && ./down.sh --config $PWD/cluster/aws/config.yaml --output $PWD/cluster/aws/
+PWD=`pwd` && bin/down.sh --config $PWD/cluster/aws/config.yaml --output $PWD/cluster/aws/
```
Comment (guineveresaenger) [inline tests/aws-testing.sh]: ./bin/down.sh

File: tests/gke-testing.sh [hunk 4]
```diff
@@ -18,14 +18,14 @@ build-scripts/update-generated-config.sh cluster/gke/config.yaml ${JOB_BASE_NAME
 
 echo "up and down"
 err=0
-PWD=`pwd` && ./up.sh --config $PWD/cluster/gke/config.yaml --output $PWD/cluster/gke/ -v '-vvv'
+PWD=`pwd` && bin/up.sh --config $PWD/cluster/gke/config.yaml --output $PWD/cluster/gke/ -v '-vvv'
```
Comment (guineveresaenger) [inline tests/gke-testing.sh]: ./bin...

File: tests/gke-testing.sh [hunk 5]
```diff
@@ -18,14 +18,14 @@ build-scripts/update-generated-config.sh cluster/gke/config.yaml ${JOB_BASE_NAME
...
 up_err=$?
 if [ $up_err -ne 0 ]; then
   err=$up_err
   echo "./up.sh failed"
 fi
 
-PWD=`pwd` && ./down.sh --config $PWD/cluster/gke/config.yaml --output $PWD/cluster/gke/
+PWD=`pwd` && bin/down.sh --config $PWD/cluster/gke/config.yaml --output $PWD/cluster/gke/
```
Comment (guineveresaenger) [inline tests/gke-testing.sh]: ./bin....

**Comment by davidewatson** [comment]
> [aws] fatal: [localhost]: FAILED! => {"failed": true, "msg": "The conditional check 'kraken_config_user.version is undefined' failed. The error was: error while evaluating conditional (kraken_config_user.version is undefined): 'kraken_config_user' is undefined\n\nThe error appears to have been in '/root/kraken/ansible/roles/kraken.config/tasks/validate-config.yaml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: Fail if still using old style configuration format\n  ^ here\n"}

File: tests/aws-testing.sh (hunk 1 above)
Comment (coffeepac) [inline tests/aws-testing.sh]: the `./` is to allow the OS to locate the script as we're providing a specific relative path with the pattern '<directory>'/'<script name>'.  In the original format `.` is the directory.  In the new format `bin` is the directory.

**Comment by coffeepac** [comment]
its a misleading error message but what's going on is that the config file wasn't being generated.  See: https://github.com/samsung-cnct/k2/issues/630 for the long term clarity fix.

**Comment by davidewatson** [comment]
Merged to unblock #102.

**Pull request merged**